src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/ScenarioRepository.kt

```diff
@@ -8,4 +9,9 @@ interface ScenarioRepository : Repository
     fun checkScenarioDescriptionExists(id: String): Unit
 
     fun getScenarios(descriptionIds: Iterable<String>): List<Scenario>
+
+    fun getScenariosForTouchstone(touchstoneVersionId: String, scenarioFilterParameters: ScenarioFilterParameters):
```

Comment:
Although the existing methods for retrieving scenarios with coverage sets are in `TouchstoneRepository`, this seemed like the better place for these. Am leaving the existing methods where they are for now, but as we introduce `ScenarioLogic` and `TouchstoneLogic` classes they will probably end up being moved/refactored.